apps/etterlatte-vedtaksvurdering-kafka/src/main/kotlin/regulering/Reguleringsforespoersel.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.etterlatte.regulering
+
+import no.nav.etterlatte.libs.common.behandling.Hendelsestype
+import no.nav.etterlatte.libs.common.behandling.Omberegningshendelse
+import no.nav.etterlatte.libs.common.behandling.RevurderingAarsak
+import no.nav.etterlatte.libs.common.logging.withLogContext
+import no.nav.etterlatte.libs.common.rapidsandrivers.correlationId
+import no.nav.etterlatte.libs.common.rapidsandrivers.eventName
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import no.nav.helse.rapids_rivers.River
+import no.nav.helse.rapids_rivers.asLocalDate
+import org.slf4j.LoggerFactory
+
+const val REGULERING_EVENT_NAME = "REGULERING" // TODO sj: Flyttes ut
+
+internal class Reguleringsforespoersel(
+    rapidsConnection: RapidsConnection,
+    private val vedtak: VedtakService
+) : River.PacketListener {
+    private val logger = LoggerFactory.getLogger(Reguleringsforespoersel::class.java)
+
+    init {
+        River(rapidsConnection).apply {
+            eventName(REGULERING_EVENT_NAME)
+            validate { it.requireKey("sakId") }
```

Comment:
Dette vil jo fungere fint for ei gitt sak, men mekanismen for å finne alle aktuelle saker kjem kanskje i neste runde?

Reply:
Ja, jeg tror vi må bestemme oss hvordan vi ser for oss å "kickstarte" reguleringen, og hvem som har ansvaret for det. 😊